templates/v3/includes/_hero_home.html

```
@@ -13,6 +13,7 @@
     hero_image_url    (optional) — Image URL for the right side. If omitted, image block is not shown.
     hero_image_url_light (optional) — Image URL for light mode. If omitted, falls back to hero_image_url.
     hero_image_url_dark  (optional) — Image URL for dark mode. If omitted, only the light image is used.
+    hero_image_url_mobile (optional) — Art-directed image swapped in at <=767px (e.g. a tighter crop). If omitted, the desktop image is used at all widths.
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)